Serializer/BaseSerializerHandler.php

```diff
@@ -27,6 +27,11 @@
     protected $manager;
 
     /**
+     * @var array
+     */
+    private static $formats = array('json', 'xml', 'yml');
```

Comment:
You're right, it cannot be changed, because it is requested by JMS serializer : http://jmsyst.com/libs/serializer/master/handlers